ts/src/kucoin.ts

```diff
@@ -1930,8 +1930,12 @@ export default class kucoin extends Exchange {
         const stopTriggered = this.safeValue (order, 'stopTriggered', false);
         const isActive = this.safeValue (order, 'isActive');
         let status = undefined;
-        if (isActive === true) {
-            status = 'open';
+        if (isActive !== undefined) {
```

Comment:
@ndubel I think this fixes it, what do you think?

Reply:
(createOrder does not have the `isActive` value, `fetchOpenOrders` returns it as `true` and `fetchClosedOrders` returns it as `false`)

Reply:
@ndubel going to merge this, let me know if anything

Reply:
thank you, I think all good